accessibilité/l-equipe-a-identifie-ses-problematiques-d-accessibilite-specifiques.md

```diff
@@ -2,12 +2,11 @@
 
 ## Description
 
-Décrivez ici les problématiques spécifiques que l'équipe a
-identifiées, telles que la cartographie, les contenus simplifiés, la
-dataviz, et les applications métier.
+L'équipe a identifié les enjeux d'accessibilité spécifiques, qui sortent du cadre du RGAA, tels que la cartographie, les contenus simplifiés, les applications mobiles, la
```

Comment:
est-ce qu'on peut développer un peu sur "qui sortent du cadre du RGAA" ? c'est pas hyper évident du pourquoi de cette règle (même moi je suis pas sûr de comprendre – le RGAA ne couvre pas certains trucs du coup faut s'en remettre au WCAG ?)

Reply:
@freesteph j'ai essayé de clarifier et de rendre ça plus clair avec des exemples. Is it better?

Reply:
beaucoup mieux, merci beaucoup